generator/api/pom.xml

```diff
@@ -32,10 +32,10 @@
 
   <properties>
     <!-- Default images used in Java based generators -->
-    <generator.java.docker.upstream>fabric8/java-alpine-openjdk8-jdk:1.2.0</generator.java.docker.upstream>
-    <generator.java.s2i.upstream>fabric8/s2i-java:1.3.4</generator.java.s2i.upstream>
-    <generator.java.docker.redhat>jboss-fuse-6/fis-java-openshift:2.0</generator.java.docker.redhat>
-    <generator.java.s2i.redhat>jboss-fuse-6/fis-java-openshift:2.0</generator.java.s2i.redhat>
+    <generator.java.docker.upstream>fabric8/java-alpine-openjdk8-jdk:${generator.java.docker.upstream.version}</generator.java.docker.upstream>
```

Comment:
The question is which we want to extract the version number separately ? We could move the whole properties here into the parent.xml, too. tbh I dont see yet the value of this extra level of indirection.